lib/seattleflu/api/routes.py

```diff
@@ -127,3 +127,30 @@ def receive_redcap_det(*, session):
     datastore.store_redcap_det(session, json.dumps(document))
 
     return "", 204
+
+
+@api_v1.route("/shipping/augur-build-metadata", methods = ['GET'])
+@authenticated_datastore_session_required
+def get_metadata(session):
+    """
+    Export metadata needed for SFS augur build
+    """
+    LOG.debug("Exporting metadata for SFS augur build")
+
+    metadata = datastore.export_metadata_for_augur_build(session)
+
+    return jsonify(metadata)
```

Comment:
I think it'd be great to send this (and the other endpoint's response) as NDJSON instead of a JSON array.

In particular, this would let us stream it back from the database to the API client instead of fetching it all into memory before sending the response.

Reply:
Tried my best with this following the flask streaming [docs](https://flask.palletsprojects.com/en/1.1.x/patterns/streaming/). 😬 
We can talk in person if this isn't what you were thinking.

Reply:
Updated according to our talk in person. Now should be streaming JSONs from the database to the client.

Reply:
It works like a charm. :-)